api/src/services/submission-service.test.ts

```diff
@@ -40,7 +40,7 @@ describe('SubmissionService', () => {
     });
   });
 
-  describe('insertSubmissionRecordWithPotentialConflict', () => {
+  describe('insertSubmissionRe', () => {
```

Comment:
You dropped this ✋`cordWithPotentialConflict`

Reply:
Weird, fixed.